Server/src/main/java/com/bbd/BeanServer/controller/TagController.java

```diff
@@ -28,7 +28,7 @@ ResponseEntity<?> returnAll() {
     return tags.isEmpty() ? ResponseEntity.notFound().build() : ResponseEntity.ok(tags);
   }
 
-  @GetMapping("/tag/find")
+  @PostMapping("/tag/find")
   ResponseEntity<?> returnSpecific(@RequestBody Tag request) {
```

Comment:
also kinda a weird way to find a tag if i find it correctly , could pass the tag id as a request param instead of passing in object, not exactly using a post request correctly

Reply:
I agree to all of these comments haha.

I've kind of set up a way to do it and it was easier to just keep doing it like this